codes/quantum/qubits/permutation_invariant.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: permutation_invariant
+physical: qubits
+logical: qubits # also qudits
+
+name: 'Permutation Invariant'
+
+introduced: '\cite{arXiv:quant-ph/0304153}'
+
+description: 'Codes which are stabilized by the symmetric group \(S_n\) on \(n\) elements, in a generalization of stabilizer codes to binary codes utilizing (non-abelian) group actions (in particular, \(S_n\) is non-abelian).'
+
+protection: 'Depends on the family. One family (parameterized by \(t\)) protects against arbitrary weight \(t\) qubit errors, another can approximately correct spontaneous decay errors. \cite{arXiv:1302.3247} \cite{doi:10.1103/PhysRevA.93.042340}. Another can protect against amplitude damping \cite{doi:10.1109/TIT.2019.2956142}. Another yet can protect against (multiple) deletion errors \cite{doi:2102.02494} TODO:CITE' # TODO: I would like to cite \cite{doi:2102.03015} (https://arxiv.org/abs/2102.03015) but I keep getting the error 'requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.crossref.org/works/2102.03015/transform/application/vnd.citationstyles.csl+json'
+
+features:
+
+  rate: '\(1 / (2t + 1)\)'
+
+  encoders: 
+   - 'With quantum harmonic oscillators (superconducting charge qubits in a ultrastrong coupling regime) in \(O(N)\) as in \cite{doi:10.1103/PhysRevA.99.012335}. Can be done in \(O(N^2)\) steps using quantum circuits TODO:CITE, or using geometric phase gates in \(O(N)\) \cite{doi:1908.01120}.' # TODO: I would like to cite \cite{doi:1904.07358} (https://arxiv.org/abs/1904.07358) but I keep getting the error 'requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.crossref.org/works/1904.07358/transform/application/vnd.citationstyles.csl+json'
+
+  decoders:
+   - 'For a family of codes, using projection, probability amplitude rebalancing, and gate teleportation can be done in \(O(N^2)\) \cite{arXiv:2102.02494}.'
+
+notes:
+  - 'can be constructed using real polynomials for high-dimensional qudit spaces \cite{doi:10.1016/j.laa.2017.06.031}'
+
+relations:
+  parents:
+    - code_id: stabilizer
```

Comment:
add quantum_cyclic as cousin, mentioning that cyclic group is subgroup of stabilizer group